### PR TITLE
Updating Topology Aware Hints docs for 1.23

### PR DESCRIPTION
**Pull request by robscott**
This updates our documentation for Topology Aware Hints to account for graduation to beta in 1.23.

Enhancement Issue: https://github.com/kubernetes/enhancements/issues/2433

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -187,6 +187,7 @@ different Kubernetes components.
 | `SuspendJob` | `false` | Alpha | 1.21 | 1.21 |
 | `SuspendJob` | `true` | Beta | 1.22 | |
 | `TopologyAwareHints` | `false` | Alpha | 1.21 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:189]:
```suggestion
| `TopologyAwareHints` | `false` | Alpha | 1.21 | 1.22 |
```

Comment (robscott) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Updated, thanks!

**Comment by kbhawkey** [comment]
Thanks @robscott .
/lgtm

**Pull request merged**